app/helpers/questions_helper.rb

```diff
@@ -0,0 +1,6 @@
+module QuestionsHelper
+
+  def sort_answer_by_most_votes(question)
```

Comment:
This would be better as a class method on the answer model
Should be possible to do with SQL / Arel / Active Record